lib/parity/environment.rb

```diff
@@ -37,8 +41,14 @@ def backup
     end
 
     def deploy
-      if deploy_to_heroku && run_migration?
+      pending_migrations = run_migration?
+
+      deployed = deploy_to_heroku
```

Comment:
Ack, this doesn't work because if `deploy_to_heroku` fails, we want to return an exit status of `1`.